test/lint/lint-submodule.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+"""
+This script checks for git modules
+"""
+
+import subprocess
+import sys
+
+def get_submodules_list():
+    command = [
+            'git',
+            'submodule',
+            'status',
+            '--recursive',
+    ]
+    return subprocess.check_output(command).decode('utf-8').splitlines()
+
+def main():
+    submodules_list = get_submodules_list()
+    if submodules_list:
+        print("These submodules were found, delete them:")
+        for submodule in submodules_list:
```

Comment:
what is the point of first splitting by newline and then joining the array by newline again when printing? Just leave it as string?

Reply:
I agree, that was unnecessary.  Updated the PR. Thanks for pointing it out !